lib/queue/producer.ts

```diff
@@ -42,7 +33,15 @@ export interface ProducerOptions {
 export interface ProducerResults {
 	error: boolean;
 	timestamp: string;
-	data: ExecuteContract['data']['payload']['data'];
+	data:
```

Comment:
You're probably right, I was doing this over the weekend trying to get things to build and pass tests, so there's probably some rough edges. 💦 Are you referring to https://github.com/product-os/autumndb/blob/master/lib/types/contracts/card.ts#L11 when you say "anyData"?

Reply:
@ramirogm I actually went ahead and updated it to be literally an `any` type as we don't know the shape its going to be in. Seems like a fairly valid use of the `any` type in my opinion, although I usually try and stay away from it.

Reply:
Fwiw an `unknown` type can be used for a similar case where it can be anything but requires you to do type checks to use it (eg `typeof x === ...`, `'...' in x`, etc)

Reply:
@Page- Thanks, I'll try it out before merging/deploying.